cnf-tests/testsuites/e2esuite/sctp/sctp.go

```diff
@@ -298,11 +304,15 @@ func startServerPod(node, namespace string, networks ...string) *k8sv1.Pod {
 
 	var res *k8sv1.Pod
 	By("Fetching the server's ip address")
+	var timeout time.Duration = 3
```

Comment:
Can you give timeout a better name? In this case, something like podRunningTimeout or podRunningWaitingTime

Reply:
Dpme